<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AggiornaVoti</class>
 <widget class="QWidget" name="AggiornaVoti">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Aggiorna Voto</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Giornata</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelGiornata">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Giocatore</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelGiocatore">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Giocatore</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxVoto">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>94</y>
     <width>56</width>
     <height>25</height>
    </rect>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="singleStep">
    <double>0.500000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Voto</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>141</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Cartellini</string>
   </property>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ammonito</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Espulso</string>
    </property>
   </widget>
  </widget>
  <widget class="QSpinBox" name="spinBoxGoal">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>114</y>
     <width>56</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Goal</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxRigoriSe">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>140</y>
     <width>56</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>146</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Rigori Segnati</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxRigoriSb">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>164</y>
     <width>56</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>170</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Rigori Sbagliati</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>196</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Autogoal</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxAutogoal">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>190</y>
     <width>56</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>250</y>
     <width>170</width>
     <height>26</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AggiornaVoti</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AggiornaVoti</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>264</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
